src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValue.cs

```diff
@@ -115,6 +116,11 @@ public static TimeValue New(TimeSpan value)
             return new TimeValue(IRContext.NotInSource(FormulaType.Time), value);
         }
 
+        public static ColorValue New(Color value)
```

Comment:
>Color [](http://example.com/codeflow?start=37&length=5)

One concern - is this taking a dependency on a new assembly? Color is defined in System.Drawing namespace; although looks like in netstandard.dll, so maybe we're ok.

Reply:
And I guess we already hit this for ColorValue anyways, so not a new concern.

Reply:
Yes, correct. This is also a supported primitive with an existing dependency on System.Drawing.